ports/devpaul123-thread-pool/portfile.cmake

```
@@ -0,0 +1,26 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO DeveloperPaul123/thread-pool
+    REF ${VERSION}
+    SHA512 4ab6cbb2f23dbfe46d04d117173b9df3f07355752801fc83a371481a4ef3f3dfe52b48c3bc4f40d75a367e4f00d80ddb29c061adae2b6829a591e33a276984c6
+    HEAD_REF master
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DTP_BUILD_TESTS=OFF
+        -DTP_BUILD_EXAMPLES=OFF
+        -DTP_BUILD_BENCHMARKS=OFF
+        -DFETCHCONTENT_FULLY_DISCONNECTED=OFF
+)
+
+vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(
+    PACKAGE_NAME ThreadPool
+    CONFIG_PATH lib/cmake/ThreadPool-${VERSION}
+)
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/" "${CURRENT_PACKAGES_DIR}/lib")

```

Header-only ports can simply start with `set(VCPKG_BUILD_TYPE release) # header-only`, then this is reduced to
```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib")

```